examples/configurations/private-flink-quickstart/main.tf

```diff
@@ -7,7 +7,7 @@ terraform {
           }
     confluent = {
       source  = "confluentinc/confluent"
-      version = "1.81.0"
+      version = "2.16.0"
```

Comment:
This one won't work as I can see there's SR resource in here: https://github.com/confluentinc/terraform-provider-confluent/blob/master/examples/configurations/private-flink-quickstart/main.tf#L195

We'll need to use https://registry.terraform.io/providers/confluentinc/confluent/latest/docs/guides/version-2-upgrade to patch it.

Reply:
I went ahead and patched the example 🫡